exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -136,13 +136,18 @@ func (e *exporter) serializeMetrics(md pdata.Metrics) []string {
 				metricLines, err := serialization.SerializeMetric(e.settings.Logger, e.cfg.Prefix, metric, e.defaultDimensions, e.staticDimensions, e.prevPts)
 
 				if err != nil {
-					e.settings.Logger.Sugar().Errorf("failed to serialize %s %s: %s", metric.DataType().String(), metric.Name(), err.Error())
+					e.settings.Logger.Sugar().Errorw(
+						"failed to serialize",
+						"datatype", metric.DataType().String(),
```

Comment:
TIL, I wasn't away you can just add the key pairs yourself. Really cool to know.

Reply:
yeah `logger.Sugar().Errorw(message, ...keys)` is the same as `logger.Sugar().With(...keys).Error(message)`